variables.md

```diff
@@ -1,204 +1,195 @@
 # VARIABLES
 
-A variable is a container for a data which provides a way of labeling data with a descriptive name. Variables label and store data in memory,update information stored in it and Reference or “get” information stored in it.
-
-It is important to distinguish that variables are not values; they contain values and represent them with a name.
-
+A variable is a container for storing data. It is used to label data with a descriptive name. Variables are not values instead they contain values and represent them with a name.
 
 ## Creating a Variable
 
-### ```var```
+### `var`
 
- Before the introduction of ES6 version of javascript, Programmers use the ```var``` keyword to create a variable.
+ The `var` keyword to create a variable before the introduction of `const` and `let` in ES6.
```

Comment:
I don't think this sentence is clear enough

Reply:
I meant to say "The `var` keyword was used in ES5 to create a variable before the introduction of `const` and `let` in ES6."